Modifications to configure.in:
- set the WITH_ICU flag unconditionally
- only output files we use

Index: libxml/configure.in
===================================================================
--- libxml.orig/configure.in	2010-07-09 15:00:21.600113911 -0700
+++ libxml/configure.in	2010-07-09 15:02:50.299108047 -0700
@@ -1229,6 +1229,9 @@
 fi
 AC_SUBST(WITH_OUTPUT)
 
+WITH_ICU=1
+AC_SUBST(WITH_ICU)
+
 WITH_ICONV=0
 if test "$with_iconv" = "no" ; then
     echo Disabling ICONV support
@@ -1456,7 +1459,7 @@
 ln -s Copyright COPYING
 
 # keep on one line for cygwin c.f. #130896
-AC_OUTPUT(libxml2.spec:libxml.spec.in Makefile include/Makefile include/libxml/Makefile doc/Makefile doc/examples/Makefile doc/devhelp/Makefile example/Makefile python/Makefile python/tests/Makefile xstc/Makefile include/libxml/xmlversion.h xml2-config libxml-2.0.pc libxml-2.0-uninstalled.pc python/setup.py)
+AC_OUTPUT(include/libxml/xmlversion.h xml2-config)
 
-chmod +x xml2-config python/setup.py
+chmod +x xml2-config
 echo Done configuring
